src/ConfigAggregator.php

```diff
@@ -231,4 +261,42 @@ private function cacheConfig(array $config, $cachedConfigFile)
             var_export($config, true)
         ));
     }
+
+    /**
+     * @param array $processors
+     * @param array $config
```

Comment:
These tags are useless, it's clear enough because of function signature. Can be removed.